client/my-sites/stats/stats-purchase/styles.scss

```
@@ -17,6 +17,7 @@
 	.stats-purchase-wizard {
 		font-family: $font-sf-pro-text;
 
+		li.stats-purchase-wizard__benefits--included,
```

```suggestion
		li.stats-purchase-wizard__benefits-item--included,
```
Could we use another dedicated class name for benefits items to avoid naming confusion?

---

Good point. Addressed in 30f6152c44